### ISSUE TITLE
Windows Antivirus Software

### ISSUE THREAD

**Issue by Lycaner88** [WeakAuras/WeakAuras2#3763]
### Description

Then i want to update i get always the message that a trojaner is in the newest update

### WeakAuras Version

4.0.3

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Lua Error

Trojaner

### Reproduction Steps

Trojaner

### Screenshots

![grafik](https://user-images.githubusercontent.com/111744883/185902181-8d8361eb-f9e8-4917-b336-f0b0e7e52485.png)

**Comment by Stanzilla** [comment]
Yeah that's a false positive, you can report it to Microsoft if you want
